Scottish-Tech-Army/Volunteer-app
# SVA-346 | Expo setup fixes

- [Description](#description)
  - [Type of change](#type-of-change)
- [Testing locally](#testing-locally)
- [Checklist](#checklist)

# Description

- Removed old `ios` file no longer needed, added app `android` and `ios` directories to gitignore, to help make sure no-one accidentally adds these to the repo / make the changeover a little easier for devs in the team
- Updated the API tunnelling command to keep the connection alive, and README instructions related to this

Fixes # SVA-346

## Type of change


# Testing locally

To test the updated API tunnelling command:

- Follow the instructions in the updated README to install `autossh`
- In one terminal window in the `api` directory, run `npm start`
- In a second terminal window in the `api` directory, run `npm run tunnel` -- this should say something like `Forwarding HTTP traffic from https://xxxxxxxxx.serveo.net`

# Checklist

- [] I have addressed accessibility, if needed
- [] I have checked the app in dark mode, if making front-end design changes
- [] I have added tests that prove my fix is effective or that my feature works
